interfaces/SealableMock.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
This interface describes 1 aspect of contract, it is not a mock, rename it to IPausable.json or ISealable.json